modules/installation-configuration-parameters.adoc

```diff
@@ -1442,7 +1442,7 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
 
 |`platform.gcp.defaultMachinePlatform.osDisk.diskType`
 |The link:https://cloud.google.com/compute/docs/disks#disk-types[GCP disk type].
-|Either the default `pd-ssd` or the `pd-standard` disk type. The control plane nodes must be the `pd-ssd` disk type. Compute nodes can be either type.
+|The default disk type for all machines. The control plane nodes must be the `pd-ssd` disk type. Compute nodes can be the `pd-ssd`, `pd-balanced`, or `pd-standard` disk types.
```

Comment:
For consistency between sentences, consider removing "the" before "control plane nodes".

I'm assuming that compute nodes cannot have more than one disk type. Consider rewriting to something like the following:

```adoc
Compute nodes can be a `pd-ssd`, `pd-balanced`, or `pd-standard` disk type.
```

Reply:
I went with "use":
```
Compute nodes can use the `pd-ssd`, `pd-balanced`, or `pd-standard` disk types.
```